crates/symbolicator/src/services/symbolication.rs

```diff
@@ -571,17 +571,13 @@ fn object_info_from_minidump_module_rust_minidump(
     ty: ObjectType,
     module: &MinidumpModule,
 ) -> RawObjectInfo {
-    let code_id = module.code_identifier().to_string();
+    let code_id = module.code_identifier();
 
     RawObjectInfo {
         ty,
-        // TODO: shouldn't this be None if code_id is empty?
-        code_id: Some(code_id),
+        code_id: Some(code_id.to_string()),
```

Comment:
nit: it doesn't look like `code_id` gets reused anywhere, so this could be a one-liner like the other fields,

`code_id: Some(code_id.code_identifier().to_string()`